ERCS/erc-7629.md

```diff
@@ -0,0 +1,296 @@
+     */
+    function owned(address owner) external view returns (uint256[] memory);
+
+    /**
+     * @dev Returns the address that owns a specific ERC-721 token.
+     * @param id The token ID.
+     */
+    function ownerOf(uint256 tokenId) external view returns (address erc721Owner);
+
+    /**
+     * @dev Returns the URI for a specific ERC-721 token.
+     * @param id The token ID.
+     */
+    function tokenURI(uint256 tokenId) external view returns (string memory);
+
+    /**
+     * @dev Approve or disapprove the operator to spend or transfer all of the sender's tokens.
+     * @param spender The address of the spender.
+     * @param amountOrId The amount of ERC-20 tokens or ID of ERC-721 tokens.
+     */
+    function approve(
+        address spender,
+        uint256 amountOrId
+    ) external returns (bool);
+
+    /**
+     * @dev Approve the spender to spend the specified amount of ERC-20 tokens.
+     * @param spender The address of the spender.
+     * @param value The amount of ERC-20 tokens to approve.
+     */
+    function erc20Approve(
+        address spender,
+        uint256 value
+    ) external returns (bool);
+
+    /**
+     * @dev Approve the operator to spend the specified ERC-721 token.
+     * @param spender The address of the spender.
+     * @param id The ID of the ERC-721 token to approve.
+     */
+    function erc721Approve(address spender, uint256 tokenId) external;
+
+    /**
+     * @dev Set or unset the approval of an operator for all tokens.
+     * @param operator The address of the operator.
+     * @param approved The approval status.
+     */
+    function setApprovalForAll(address operator, bool approved) external;
+
+    /**
+     * @dev Transfer ERC-20 tokens or ERC-721 token from one address to another.
+     * @param from The address to transfer ERC-20 tokens or ERC-721 token from.
+     * @param to The address to transfer ERC-20 tokens or ERC-721 token to.
+     * @param amountOrId The amount of ERC-20 tokens or ID of ERC-721 tokens to transfer.
+     */
+    function transferFrom(
+        address from,
+        address to,
+        uint256 amountOrId
+    ) external returns (bool);
+
+    /**
+     * @dev Transfer ERC-20 tokens from one address to another.
+     * @param from The address to transfer ERC-20 tokens from.
+     * @param to The address to transfer ERC-20 tokens to.
+     * @param value The amount of ERC-20 tokens to transfer.
+     */
+    function erc20TransferFrom(
+        address from,
+        address to,
+        uint256 value
+    ) external returns (bool);
+
+    /**
+     * @dev Transfer ERC-721 token from one address to another.
+     * @param from The address to transfer ERC-721 token from.
+     * @param to The address to transfer ERC-721 token to.
+     * @param id The ID of the ERC-721 token to transfer.
+     */
+    function erc721TransferFrom(
+        address from,
+        address to,
+        uint256 tokenId
+    ) external;
```

Comment:
```suggestion
    ) external payable;
```
From the original `transferForm` function of ERC721:
`function transferFrom(address _from, address _to, uint256 _tokenId) external payable;`